### PR TITLE
Fix for 2 gcc-8.3.0 new PR builds

### PR DESCRIPTION
**Pull request by fryeguy52**
@trilinos/framework 

## Motivation
fix new gcc-8.3.0 sems-v2 builds for PR testing

### CODE REVIEW

**Review by DaveBeCoding — APPROVED** [review]

File: packages/framework/ini-files/config-specs.ini [hunk 1]
```diff
@@ -2257,24 +2257,24 @@ use KOKKOS-ARCH|NO-KOKKOS-ARCH
 use USE-ASAN|NO
 use USE-COMPLEX|NO
 use USE-FPIC|NO
-use USE-MPI|NO
+use USE-MPI|YES
```
Comment (sebrowne) [inline packages/framework/ini-files/config-specs.ini:2260]: This is likely what caused the MPI build issue.

File: packages/framework/ini-files/config-specs.ini [hunk 2]
```diff
@@ -2306,6 +2306,8 @@ opt-set-cmake-var Trilinos_ENABLE_Fortran OFF    BOOL         : OFF
 opt-set-cmake-var Trilinos_ENABLE_COMPLEX_DOUBLE BOOL         : ON
 opt-set-cmake-var CMAKE_CXX_FLAGS                STRING       : -Wall -Wno-clobbered -Wno-vla -Wno-pragmas -Wno-unknown-pragmas -Wno-parentheses -Wno-unused-local-typedefs -Wno-literal-suffix -Wno-deprecated-declarations -Wno-misleading-indentation -Wno-int-in-bool-context -Wno-maybe-uninitialized -Wno-class-memaccess -Wno-nonnull-compare -Wno-address -Wno-inline -Wno-unused-but-set-variable -Wno-unused-variable -Wno-unused-label -Werror -DTRILINOS_HIDE_DEPRECATED_HEADER_WARNINGS
 opt-set-cmake-var TPL_ENABLE_ParMETIS            BOOL FORCE   : OFF
+opt-set-cmake-var TPL_ENABLE_Pnetcdf             BOOL FORCE   : OFF
+opt-set-cmake-var TPL_Netcdf_LIBRARIES           STRING FORCE : -L${NETCDF_C_LIB|ENV};${NETCDF_C_LIB|ENV}/libnetcdf.a;${TPL_HDF5_LIBRARIES|CMAKE}
```
Comment (sebrowne) [inline packages/framework/ini-files/config-specs.ini:2310]: And this likely caused the issues with the nompi build.

**Review by sebrowne — APPROVED** [review]

File: packages/framework/ini-files/config-specs.ini (hunk 1 above)
Comment (fryeguy52) [reply packages/framework/ini-files/config-specs.ini:2260]: Yep

File: packages/framework/ini-files/config-specs.ini (hunk 2 above)
Comment (fryeguy52) [reply packages/framework/ini-files/config-specs.ini:2310]: exactly

**Pull request merged**